client/src/send_and_confirm_transactions_in_parallel.rs

```diff
@@ -349,10 +348,25 @@ async fn confirm_transactions_till_block_height_and_resend_unexpired_transaction
                     .iter()
                     .filter(|x| block_height < x.last_valid_block_height)
                     .map(|x| x.serialized_transaction.clone())
-                    .collect();
-                let _ = tpu_client
-                    .try_send_wire_transaction_batch(txs_to_resend_over_tpu)
-                    .await;
+                    .collect::<Vec<_>>();
+                let num_txs_to_resend = txs_to_resend_over_tpu.len();
+                let message = if tokio::time::timeout(
+                    Duration::from_secs(5),
```

Comment:
should be `BLOCKHASH_REFRESH_RATE`?

Reply:
I thought about it, but worried it would overload the constant to use it in a different setting, since this number was chosen arbitrarily, and not because of `BLOCKHASH_REFRESH_RATE`. But if you think it's clearer, I'm happy to change it!

Reply:
I thought you just overlooked this constant. It looks confusing because it has the same value and because it is unnamed while others are named. So I think there should be comment saying that this constant has been empirically chosen or a different constant

Reply:
I added in a comment, let me know how it looks!